static/js/github-configuration.js

```diff
@@ -26,6 +26,7 @@ $('.delete-link').click(function (event) {
     if (data.err) {
       return console.log(data.err)
     }
+		window.opener.postMessage({moduleKey: "github-post-install-page"}, window.location.origin);
```

Comment:
/nit
maybe add a messageType along with the moduleKey, with 'goto_module_page'.
purely for code readability

Reply:
I thought about adding it, but felt like its of not much value at this point. Will leave it for now...